app/src/main/java/com/ncnf/map/MapHandler.java

```diff
@@ -55,131 +55,131 @@ public MapHandler(Activity context, GoogleMap mMap, EventProvider eventProvider,
-
-            }
-        });
-    }
-
-    //Creates the behavior for when clicking on a suggestion
-    private void createSuggestionsClickListener(MaterialSearchBar materialSearchBar, PlacesClient placesClient){
-        materialSearchBar.setSuggestionsClickListener(new SuggestionsAdapter.OnItemViewClickListener() {
-            @Override
-            public void OnItemClickListener(int position, View v) {
-                if (position >= predictionList.size())
-                    return;
-                AutocompletePrediction selection = predictionList.get(position);
-                //Stores the address as a human-readable String in 'suggestion'
-                String suggestion = materialSearchBar.getLastSuggestions().get(position).toString();
-                materialSearchBar.setText(suggestion);
-
-                //Because apparently just putting mSB.clearSuggestions() does not work
-                new Handler().postDelayed(materialSearchBar::clearSuggestions, 1000);
-
-                //Hides the keyboard
-                InputMethodManager input = (InputMethodManager) context.getSystemService(INPUT_METHOD_SERVICE);
-                if (input != null)
-                    input.hideSoftInputFromWindow(materialSearchBar.getWindowToken(), InputMethodManager.HIDE_IMPLICIT_ONLY);
-
-                //Prepares the conversion from prediction to LatLng
-                final String placeId = selection.getPlaceId();
-                List<Place.Field> placeFields = Arrays.asList(Place.Field.LAT_LNG);
-
-                //Converts prediction to LatLng and updates markers if successful, prints out errors if not
-                FetchPlaceRequest fetchPlaceRequest = FetchPlaceRequest.builder(placeId, placeFields).build();
-                placesClient.fetchPlace(fetchPlaceRequest).addOnSuccessListener(fetchPlaceResponse -> {
-                    Place place = fetchPlaceResponse.getPlace();
-                    Log.i("Place fetching", "Place found: " + place.getName());
-                    LatLng placeLatLng = place.getLatLng();
-                    if (placeLatLng != null) {
-                        userPosition = placeLatLng;
-                        update_markers();
-                    }
-                }).addOnFailureListener(e -> {
-                    if (e instanceof ApiException){
-                        ApiException apiException = (ApiException) e;
-                        apiException.printStackTrace();
-                        int statusCode = apiException.getStatusCode();
-                        Log.i("Place fetching", "Place not found: " + e.getMessage());
-                        Log.i("Place fetching", "Status code: " + statusCode);
-                    }
-                });
-            }
-
-            @Override
-            public void OnItemDeleteListener(int position, View v) {
-
-            }
-        });
-    }
+//    public void createOnSearchActionListener(MaterialSearchBar materialSearchBar){
```

Comment:
Wouldn't it be better to remove the huge commented blocks of code? Especially since it's been copied into the SearchBarHandler

Reply:
Same thing as before, wanted to be sure.